next/lib/strapi/strapiImage.ts

```diff
@@ -1,5 +1,13 @@
+import { unstable_noStore as noStore } from 'next/cache';
+
 export function strapiImage(url: string): string {
+  noStore();
   if (url.startsWith("/")) {
+
+    if (!process.env.NEXT_PUBLIC_API_URL && document.location) {
```

Comment:
I think we should enter here only on Strapi demos. WDYT?

```suggestion
    if (!process.env.NEXT_PUBLIC_API_URL && document?.location.host.endsWith(".strapidemo.com")) {
```

Reply:
Absolutely!